fixtures/response/user_response_suggestions.py

```diff
@@ -2,7 +2,7 @@
 
 create_suggestion_question = '''
 mutation{
-  createResponse(questionId:3, roomId:1, textArea:"Any other suggestion") {
+  createResponse(responses: [{questionId:3, textArea:"Any other suggestion"}], roomId:1) { # noqa
```

Comment:
@kampkelly you can do away with the `# noqa` on lines `5` `33` `46` by moving `responses: [...`] on a new line.

Reply:
Thanks. It has been done.